src/tests/ftest/telemetry/telemetry_pool_metrics.py

```diff
@@ -161,17 +151,29 @@ def test_telemetry_pool_metrics(self):
         # collect second set of pool metric data after read/write
         metrics_end = self.get_metrics(metric_names)
 
+        # Compute the number of operations done
         metrics = {}
         for name in metric_names:
             metrics[name] = metrics_end[name] - metrics_init[name]
             self.log.debug(
                 "Successfully retrieve metric: %s=%d (init=%d, end=%d)",
                 name, metrics[name], metrics_init[name], metrics_end[name])
 
-        # perform verification check
+        # NOTE DAOS-14220: Check if networking errors occurred during the test.  If yes, we skip the
```

Comment:
Why does networking error occur? Isn't it better to skip this test until the root cause of the error is fixed?

Reply:
From my understanding, these errors are transient networking errors happening with the networking fabric and thus could not be predicted.  Thus, this kind of errors could always happen.  Initially, I tried to deal with them with updating the expected values of the tested telemetry counters. At the end, I am not able to properly handle this and thus the better solution that I have found is to skip this test when such errors are detected.
Does it make sense ?